roles/common/tasks/monitoring.yml

```diff
@@ -64,6 +64,12 @@
               line="PATH=/opt/sensu/embedded/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/etc/sensu/plugins"
   notify: restart sensu-client
 
+- name: ensure sensu client use base_venv python
+  lineinfile: dest=/etc/default/sensu regexp="^\. {{ common.python.base_venv }}/bin/activate"
+              line=". {{ common.python.base_venv }}/bin/activate"
+  notify: restart sensu-client
+  when: openstack_install_method == 'distro' and os == 'rhel'
```

Comment:
We don't need this conditional any longer. The base venv will be setup on all OS's so we should do this on ubuntu/centos/rhel.